--- configure.ac	2020-12-03 01:08:53.000000000 +0800
+++ configure.ac	2022-02-16 06:04:06.770474753 +0800
@@ -1728,7 +1728,6 @@ dnl otherwise none of both
                 -Wno-long-long
                 -Wno-strict-aliasing
                 -Wpointer-arith
-                -Wstrict-prototypes
                 -Wwrite-strings
                 -Wunused-but-set-variable
                 -Wunsigned-char"
