diff --git a/cts/cts-cli.in b/cts/cts-cli.in
index cff4bbf..707403f 100644
--- a/cts/cts-cli.in
+++ b/cts/cts-cli.in
@@ -947,7 +947,7 @@ EOF
     if [ "$(uname)" == "FreeBSD" ]; then
         tomorrow=$(date -v+1d +"%F %T %z")
     else
-        tomorrow=$(date --date=tomorrow +"%F %T %z")
+        tomorrow=$(date --date=tomorrow +"%F %T")
     fi
 
     TMPXML=$(mktemp ${TMPDIR:-/tmp}/cts-cli.tools.xml.XXXXXXXXXX)
